Source/Classes/AnimatedImages/PINCachedAnimatedImage.m

```diff
@@ -379,12 +382,13 @@ - (CFTimeInterval)durationAtIndex:(NSUInteger)index
 
 - (BOOL)playbackReady
 {
-    static BOOL playbackReady = NO;
-    if (playbackReady == NO) {
-        [_lock lockWithBlock:^{
-            playbackReady = _playbackReady == 0;
-        }];
-    }
+    __block BOOL playbackReady = NO;
+    [_lock lockWithBlock:^{
+        if (_playbackReady == NO) {
```

Comment:
Do we need to have the `_playbackReady` instance variable at all? what about just having:

```
- (BOOL)playbackReady
    __block BOOL playbackReady = NO;
    [_lock lockWithBlock:^{
       playbackReady = (_frameRenderCount == 0);
    }];
    return playbackReady;
}
```

From a cost perspective should be even cheaper as we just check the `_frameRenderCount` variable and not have to access the `_playbackReady` as well as the `_frameRenderCount` eventually? Just a thought though.